js/breadcrumbs.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Requires jQuery and Bootstrap
+ */
+namespacer('bcpl');
+
+bcpl.breadCrumbs = (function breadCrumbs($) {
+	const templates = {
+		popover: '<div class="popover breadcrumb-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>'
+	};
+	const classNames = {
+		breadCrumbContainer: 'breadcrumbs-wrapper',
+		firstBreadCrumb: 'breadcrumb-first',
+		hiddenBreadCrumbContainer: 'hidden-breadcrumb-container',
+		hiddenBreadCrumbTrigger: 'hidden-breadcrumb-trigger',
+		hiddenBreadCrumbPopover: 'hidden-breadcrumb-popover'
+	};
+
+	const selectors = {
+		breadCrumbChildren: '.breadcrumbs-wrapper a, .breadcrumbs-wrapper span'
+	};
+
+	const buildBreadCrumbHtml = (hiddenBreadCrumbTriggerClassName) => {
+		const hiddenBreadCrumbTriggerHtml = buildBreadCrumbTrigger(hiddenBreadCrumbTriggerClassName);
+		return `<div class="${classNames.hiddenBreadCrumbContainer} ${classNames.hiddenBreadCrumbPopover} breadcrumb breadcrumb-alt" data-toggle="popover" id="tip1"><div class="tooltip-arrow"></div>${hiddenBreadCrumbTriggerHtml}</div>`;
+	};
+
+	const buildBreadCrumbList = ($hiddenBreadCrumbs) => {
+		const listItems = $hiddenBreadCrumbs.toArray().map((hiddenBreadCrumbElm) => {
+			const $clonedBreadCrumb = $(hiddenBreadCrumbElm).clone();
+			$clonedBreadCrumb.removeClass('breadcrumb').removeAttr('style');
+
+			return `<li>${$clonedBreadCrumb[0].outerHTML}</li>`;
+		}).join('');
+
+		return listItems.length ? `<ul class="hidden-breadcrumb-list">${listItems}</ul>` : '';
+	};
+
+	const buildBreadCrumbTrigger = (breadCrumbTrigger) => `<span class="${breadCrumbTrigger}"><i class="fa fa-circle" aria-hidden="true"></i><i class="fa fa-circle" aria-hidden="true"></i><i class="fa fa-circle" aria-hidden="true"></i></span>`;
+
+	const isEmpty = ($elm) => !$.trim($elm.html().replace(/(?:\r\n|\r|\n|\s|&nbsp;)/g, ''));
```

Comment:
Eventually, this can be moved to a utility class.

Reply:
@mxsnyder good idea, tracked here, https://github.com/baltimorecounty/BCPL-assets/issues/184